llvm/test/CodeGen/SPIRV/scfg-add-pre-headers.ll

```diff
@@ -0,0 +1,67 @@
+; RUN: llc -mtriple=spirv-unknown-unknown -O0 %s -o - | FileCheck %s
+
+; CHECK-DAG:  [[bool:%[0-9]+]] = OpTypeBool
```

Comment:
Could you please change the check lines to match the format in other newly added LIT test files (if the same functionality can be maintained)? E.g. `%[[#bool:]]`? This will make it easier for us to do any bulk find-and-replace in the future.

Reply:
Ah thanks for pointing this out, fixed!